config/presets/testnet.go

```diff
@@ -1,73 +1,145 @@
+			MetricsPort:         1010,
+			DatabaseConnections: 16,
+			NetworkHRP:          "smtest",
 
-	conf.POST.K1 = 26
-	conf.POST.K2 = 37
-	conf.POST.K3 = 37
-	conf.POST.LabelsPerUnit = 20 * 1024 / 16 // 20 kB units
-	conf.POST.MaxNumUnits = 4
-	conf.POST.MinNumUnits = 2
+			LayerDuration:  5 * time.Minute,
+			LayerAvgSize:   50,
+			LayersPerEpoch: 288,
 
-	conf.SMESHING.CoinbaseAccount = types.GenerateAddress([]byte("1")).String()
-	conf.SMESHING.Start = false
-	conf.SMESHING.Opts.ProviderID.SetInt64(int64(initialization.CPUProviderID()))
-	conf.SMESHING.Opts.NumUnits = 2
-	conf.SMESHING.Opts.Throttle = true
+			TxsPerProposal: 700,       // https://github.com/spacemeshos/go-spacemesh/issues/4559
+			BlockGasLimit:  100107000, // 3000 of spends
 
-	conf.Beacon.FirstVotingRoundDuration = 3 * time.Minute
-	conf.Beacon.GracePeriodDuration = 10 * time.Second
-	conf.Beacon.ProposalDuration = 30 * time.Second
-	conf.Beacon.RoundsNumber = 6
-	conf.Beacon.BeaconSyncWeightUnits = 30
-	conf.Beacon.VotesLimit = 100
-	conf.Beacon.VotingRoundDuration = 50 * time.Second
-	conf.Beacon.WeakCoinRoundDuration = 10 * time.Second
+			OptFilterThreshold: 90,
 
-	return conf
+			TickSize:    666514,
+			PoETServers: []string{},
+		},
+		Genesis: &config.GenesisConfig{
+			GenesisTime: "2023-09-13T08:00:00Z",
+			ExtraData:   "0000000000000000000000c76c58ebac180989673fd6d237b40e66ed5c976ec3",
+		},
+		Tortoise: tortoise.Config{
+			Hdist:                    10,
+			Zdist:                    2,
+			WindowSize:               10000,
+			MaxExceptions:            1000,
+			BadBeaconVoteDelayLayers: 4032,
+			// 100 - is assumed minimal number of units
+			// 100 - half of the expected poet ticks
+			MinimalActiveSetWeight: 100 * 100,
+		},
+		HARE: hareConfig.Config{
+			N:               200,
+			ExpectedLeaders: 5,
+			RoundDuration:   25 * time.Second,
+			WakeupDelta:     25 * time.Second,
+			LimitConcurrent: 2,
+			LimitIterations: 4,
+		},
+		HareEligibility: eligConfig.Config{
+			ConfidenceParam: 20,
+		},
+		Beacon: beacon.Config{
+			Kappa:                    40,
+			Q:                        big.NewRat(1, 3),
+			Theta:                    big.NewRat(1, 4),
+			GracePeriodDuration:      10 * time.Minute,
+			ProposalDuration:         4 * time.Minute,
+			FirstVotingRoundDuration: 30 * time.Minute,
+			RoundsNumber:             20,
+			VotingRoundDuration:      4 * time.Minute,
+			WeakCoinRoundDuration:    4 * time.Minute,
+			VotesLimit:               100,
+			BeaconSyncWeightUnits:    800,
+		},
+		POET: activation.PoetConfig{
+			PhaseShift:        12 * time.Hour,
+			CycleGap:          2 * time.Hour,
+			GracePeriod:       10 * time.Minute,
+			RequestRetryDelay: 10 * time.Second,
+			MaxRequestRetries: 10,
+		},
+		POST: activation.PostConfig{
+			MinNumUnits:   4,
+			MaxNumUnits:   math.MaxUint32,
+			LabelsPerUnit: 1024,
+			K1:            26,
+			K2:            37,
+			K3:            37,
+			PowDifficulty: postPowDifficulty,
```

Comment:
Do we want to keep mainnet difficulty for K2pow? It will probably take more time than reading the data (the difficulty scales with the number of units, not the number of labels).

If so, consider:
```suggestion
			PowDifficulty: activation.PowDifficulty(config.MainnetConfig().PowDifficulty),
```
from `github.com/spacemeshos/post/config`

Reply:
i prefer not to init mainnet config

Reply:
What do you mean? 🤔

Reply:
thats what you suggested?

> activation.PowDifficulty(config.MainnetConfig().PowDifficulty)

Reply:
It's equivalent, just using the mainnet setting defined in `post` repo. Actually, if you want mainnet POST config, you can just use the whole `config.MainnetConfig()`.

I first asked if you really want to have the same k2pow difficulty as on mainnet. K2pow will most likely take a lot of time for smeshers that init many SU (it could take more than the 2h cycle gap).

Reply:
OK, I see it's reduced already :+1: